resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/driver/Dockerfile

```diff
@@ -15,26 +15,13 @@
 # limitations under the License.
 #
 
-FROM openjdk:8-alpine
+FROM spark-base
 
 # If this docker file is being used in the context of building your images from a Spark distribution, the docker build
 # command should be invoked from the top level directory of the Spark distribution. E.g.:
 # docker build -t spark-driver:latest -f dockerfiles/driver/Dockerfile .
 
-RUN apk upgrade --update
-RUN apk add --update bash tini
-RUN mkdir -p /opt/spark
-RUN touch /opt/spark/RELEASE
-
-ADD jars /opt/spark/jars
-ADD examples /opt/spark/examples
-ADD bin /opt/spark/bin
-ADD sbin /opt/spark/sbin
-ADD conf /opt/spark/conf
-
-ENV SPARK_HOME /opt/spark
-
-WORKDIR /opt/spark
+COPY examples /opt/spark/examples
```

Comment:
Good summary of `ADD` vs `COPY` : https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy

`ADD` is somewhat deprecated because it has complex behaviors.  Recommended best practice is to copy tar-balls and unpack them explicitly in a `RUN`